### ISSUE TITLE
Support for unix pipes for the CLI

### ISSUE THREAD

**Issue by jhpoelen** [frictionlessdata/frictionless-py#326]
Currently, I can use your nice (!) command-line tool to do:

```
$ echo -e "one\ttwo\noneValue\ttwoValue" > hello.tsv
$ goodtables hello.tsv 
DATASET
=======
{'error-count': 0,
 'preset': 'nested',
 'table-count': 1,
 'time': 0.102,
 'valid': True}

TABLE [1]
=========
{'encoding': 'utf-8',
 'error-count': 0,
 'format': 'tsv',
 'headers': ['one', 'two'],
 'row-count': 2,
 'scheme': 'file',
 'source': 'hello.tsv',
 'time': 0.005,
 'valid': True}
```

However, I cannot (yet?) do:

```
$ echo -e "one\ttwo\noneValue\ttwoValue" | goodtables 
...
```

Are you planning to support pipes?

**Comment by roll** [comment]
It looks like a good idea :+1: 

Would you like to try PRing `goodtables/cli.py`?

**Comment by jhpoelen** [comment]
Sounds good. Not sure when I'll be able to work on this, so there might be a bit of a delay. Please don't wait for me if you (or someone else) get inspired.

**Comment by roll** [comment]
MERGED into #386

**Comment by jhpoelen** [comment]
@roll thanks for moving this feature into the #386 issue .  I didn't get a chance to work on it.
